README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
@kai33 The inline task scans in all html for script tags and link tags for JS and CSS files and since the task is ran only when building the project, future developers can continue using script tags and links tags as per normal.